pyvista/core/filters/data_set.py

```diff
@@ -4154,15 +4154,20 @@ def plot_over_circular_arc_normal(
         if show:  # pragma: no cover
             plt.show()
 
-    def extract_cells(self, ind, progress_bar=False):
+    def extract_cells(self, ind, assume_unique=False, progress_bar=False):
         """Return a subset of the grid.
 
         Parameters
         ----------
-        ind : np.ndarray
+        ind : numpy.ndarray
             Numpy array of cell indices to be extracted.
 
-        progress_bar : bool, optional
+        assume_unique : bool, default: False
```

Comment:
This isn't used in this method?

Reply:
It's not and it's been removed in 8bcb10af7417fc5ab491b6922f193f7d3fd64ae5.